codenames-front-end/src/pages/Authentication/SignIn/SingIn.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react';
+import {StyledSignIn} from "./SignInStyles";
+import {StyledBlueYellowBG} from "../../../components";
+import {
+    StyledAuthenticationButton,
+    StyledAuthenticationFormTitle,
+    StyledAuthenticationFrom, StyledAuthenticationInput,
+    StyledAuthenticationInputBlock
+} from "../AuthenticationStyles";
+import useFetching from "../../../hooks/useFetching";
+import {authRequest, removeAuthToken, setAuthToken} from "../../../helper";
+import {RestConfig} from "../../../config";
+import {notify} from "../../../models";
+import {AxiosResponse} from "axios";
+import {useNavigate} from "react-router-dom";
+
+type SignInResponse = {
+    nickname: string;
+    jwtToken: string;
+    login: string;
+}
+
+const SingIn = () => {
```

Comment:
`SignIn`

Reply:
I apologize for any problems with grammar